docs/NewAPI.md

```diff
@@ -67,7 +67,7 @@ If you do not care about confirmation you can use producer as `Sink` and create
 
 ![Producer shape](./Producer-sink.png)
 
-To complete producer just complete it `In`.
+To complete producer just complete it `In`. <!-- something missing here?? -->
```

Comment:
@akauppi I want to explain how to close producer. Producer is represented as `Flow`. Flow has two ports - `In` port and `Out` port. You publish messages into `In` port and get confirmation from `Out` port. If you want to complete producer you should compete `In` port. It can be done via akka-stream API. I can provide any other clarifications :)